docs/user-guide/installandconfig.md

```diff
@@ -28,6 +28,9 @@ The following information is required during the installation process. Software
 - (If not using Docker) Zowe uses a zFS directory to contain its northbound certificate keys as well as a truststore for its southbound keys. Northbound keys are one presented to clients of the Zowe desktop or Zowe API Gateway, and southbound keys are for servers that the Zowe API gateway connects to.  The certificate directory is not part of the Zowe runtime so that it can be shared between multiple Zowe runtimes and have its permissions secured independently. 
 
 - Zowe has the following started tasks:
+
+   **Note**: API ML installation requires only `ZWESVSTC`, so you can skip `ZWESISTC` and `ZWESASTC`.
```

Comment:
I thought that `ZWESISTC` is required for x509 client side certificate authentication which APIML uses, and after PI3 will be a feature of the Zowe CLI that can route requests through APIML (that in turn calls to ZWESISTC).

Maybe there should be a markdown table that shows each STC and then what it's used for,  so you could show that APIML only need ZWESVSTC for basic auth and JSON web token authentication, .but that ZWESISTC is needed for x509.

Reply:
Thank you for reviewing the note! Unfortunately, I'm not an API ML expert but I can check with the team that is responsible for it. Maybe we can put together a table with the required started tasks.

Reply:
Including @jalel01 , maybe he can point me in the right direction here. Who would be the right person to talk to about the issue?

Reply:
Hi Igor, looks like this is resolved already - do you need any further assistance from the api ml squad?

Reply:
Thanks for getting back @jalel01 I just wanted to make sure the information is the notes is technically correct

Reply:
The current state is technically correct.

Reply:
That's great if it's OK as-is.  I was mistaken, but I thought that x.509 client certificate support from APIML required ZSS which requires cross-memory.